src/EmployeesAdministrationApp.java

```diff
@@ -151,21 +157,16 @@ private static void addEmployee(Object employee, String message) {
 				}
 			}
 		}
-		Map<String, Map<String, Object>> employeeDetailsJson = new HashMap<>();
-		employeeDetailsJson.put("employee", employeeDetails);
-		JSONObject employeeDetailsJsonObject = new JSONObject(employeeDetailsJson);
-		listOfEmployees.add(employeeDetailsJsonObject);
+		listOfEmployees.add((HashMap<String, Object>) employeeDetails);
```

Comment:
this needs to be changed

Reply:
I changed the list of employees from List to HashMap